### PR TITLE
Add Makefile

### PR DESCRIPTION
**Pull request by realloc**
Preparing for packaging locode_db.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	@echo '    make <target>'
+	@echo ''
+	@echo '  Targets:'
+	@echo ''
+	@awk '/^#/{ comment = substr($$0,3) } comment && /^[a-zA-Z][a-zA-Z0-9_-]+ ?:/{ print "   ", $$1, comment }' $(MAKEFILE_LIST) | column -t -s ':' | grep -v 'IGNORE' | sort -u
+
+# Clean up
+clean:
+	rm -f in/*
+	rm -f locode_db
```
Comment (carpawell) [inline Makefile:58]: `tmp` too?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+in/continents.geojson: $(DIRS)
+	zcat continents.geojson.gz > in/continents.geojson
+
+unlocode:
+	wget -c https://service.unece.org/trade/locode/loc221csv.zip -O tmp/loc221csv.zip
+	unzip -u tmp/loc221csv.zip -d in/
+
+locode_db: unlocode in/continents.geojson in/airports.dat in/countries.dat
+	$(NEOFSCLI) util locode generate \
```
Comment (carpawell) [inline Makefile:32]: do we know we have CLI?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	--continents in/continents.geojson \
+	--countries in/countries.dat \
+	--in in/2022-1\ UNLOCODE\ CodeListPart1.csv,in/2022-1\ UNLOCODE\ CodeListPart2.csv,in/2022-1\ UNLOCODE\ CodeListPart3.csv \
+	--subdiv in/2022-1\ SubdivisionCodes.csv \
+	--out locode_db
+
+# Print version
+version:
+	@echo $(VERSION)
+
```
Comment (carpawell) [inline Makefile:43]: redundant?

**Review by cthulhu-rider — DISMISSED** [review]
Tested: works fine

File: Makefile (hunk 2 above)
Comment (realloc) [reply Makefile:32]: Yes, we do.

**Review by carpawell — APPROVED** [review]

**Pull request merged**